### PR TITLE
Replace most VBox/HBoxContainers with BoxContainers

### PR DESCRIPTION
**Pull request by Visne**
Pain. Agony even.

### CODE REVIEW

File: Content.Client/Atmos/UI/GasAnalyzerMenu.cs [hunk 1]
```diff
@@ -195,8 +209,9 @@ public void Populate(GasAnalyzerBoundUserInterfaceState state)
             // This is the gas bar thingy
             var height = 30;
             var minSize = 24; // This basically allows gases which are too small, to be shown properly
-            var gasBar = new HBoxContainer
+            var gasBar = new BoxContainer
             {
+            	Orientation = LayoutOrientation.Horizontal,
```
Comment (mirrorcult) [inline Content.Client/Atmos/UI/GasAnalyzerMenu.cs:214]: indentation

File: Content.Client/Atmos/UI/GasAnalyzerMenu.cs [hunk 2]
```diff
@@ -178,10 +185,17 @@ public void Populate(GasAnalyzerBoundUserInterfaceState state)
...
+            {
+                Orientation = LayoutOrientation.Vertical
+            };
+            var tableVal = new BoxContainer
+            {
+                Orientation = LayoutOrientation.Vertical
+            };
+            _statusContainer.AddChild(new BoxContainer
             {
+            	Orientation = LayoutOrientation.Horizontal,
```
Comment (mirrorcult) [inline Content.Client/Atmos/UI/GasAnalyzerMenu.cs:198]: indentation

File: Content.Client/Atmos/UI/GasAnalyzerMenu.cs [hunk 3]
```diff
@@ -118,8 +124,9 @@ public GasAnalyzerWindow(GasAnalyzerBoundUserInterface owner)
                 PanelOverride = new StyleBoxFlat { BackgroundColor = Color.FromHex("#202025") },
                 Children =
                 {
-                    (_statusContainer = new VBoxContainer
+                    (_statusContainer = new BoxContainer
                     {
+                    	Orientation = LayoutOrientation.Vertical,
```
Comment (mirrorcult) [inline Content.Client/Atmos/UI/GasAnalyzerMenu.cs:129]: indentation

File: Content.Client/Atmos/UI/GasAnalyzerMenu.cs [hunk 4]
```diff
@@ -78,8 +83,9 @@ public GasAnalyzerWindow(GasAnalyzerBoundUserInterface owner)
             var fontSmall = resourceCache.GetFont("/Fonts/Boxfont-round/Boxfont Round.ttf", 10);
 
             Button refreshButton;
-            var topRow = new HBoxContainer
+            var topRow = new BoxContainer
             {
+            	Orientation = LayoutOrientation.Horizontal,
```
Comment (mirrorcult) [inline Content.Client/Atmos/UI/GasAnalyzerMenu.cs:88]: indentation

File: Content.Client/Arcade/BlockGameMenu.cs [hunk 5]
```diff
@@ -282,16 +291,20 @@ public BlockGameMenu(BlockGameBoundUserInterface owner)
 
             _highscoresRootContainer.AddChild(menuInnerPanel);
 
-            var menuContainer = new VBoxContainer()
+            var menuContainer = new BoxContainer
             {
+            	Orientation = LayoutOrientation.Vertical,
```
Comment (mirrorcult) [inline Content.Client/Arcade/BlockGameMenu.cs:296]: ok the indentation is just fucked on most of these so im just not gonna mention it further

**Review by mirrorcult — COMMENTED** [review]
is this PR going to touch XAML UIs at all? seems like more of the same stuff

**Comment by Visne** [comment]
I accidentally indented using tabs instead of spaces. Should be fixed for all files now. I will do the XAML UIs too.

**Comment by Visne** [comment]
Fixed indentation and also did XAML UI.

**Pull request merged**